02_production.ipynb

```diff
@@ -2,9 +2,18 @@
  "cells": [
```

Comment:
<p>As well as adding DDG search, should the BING one here be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastbook/pull/516/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>